CLAUDE.md

```
@@ -0,0 +1,56 @@
+# Commands & Guidelines for markdown_lab
+
+## Build & Test Commands
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)